install-and-configure/install/multi-cluster/long-term-storage-configuration/long-term-storage-aws.md

```diff
@@ -3,22 +3,22 @@ AWS Multi-Cluster Storage Configuration
 
 ## AWS/S3 Federation
 <a name="overview"></a>
-Kubecost uses a shared storage bucket to store metrics from clusters (aka `durable storage`) in order to provide a single-pane-of-glass for viewing cost across many clusters. Multi-cluster is an enterprise feature of Kubecost.
+Kubecost uses a shared storage bucket to store metrics from clusters, known as durable storage, in order to provide a single-pane-of-glass for viewing cost across many clusters. Multi-cluster is an enterprise feature of Kubecost.
 
 There are multiple methods to provide Kubecost access to an S3 bucket. This guide has two examples:
 
 1. Using a Kubernetes secret
-1. Attaching an AWS Identity and Access Management (IAM) role to the service account used by Prometheus
+2. Attaching an AWS Identity and Access Management (IAM) role to the service account used by Prometheus
 
 Both methods require an S3 bucket. Our example bucket is named `kc-thanos-store`.
 
 This is a simple S3 bucket with all public access blocked. No other bucket configuration changes should be required.
 
-Once created, add an IAM policy to access this bucket ([steps](/aws-service-account-thanos.md)).
+Once created, add an IAM policy to access this bucket. This is covered in our [AWS Thanos IAM Policy](/aws-service-account-thanos.md) doc.
 
 ## Method 1: Kubernetes Secret Method
 <a name="secret"></a>
-To use the Kubernetes secret method for allowing access, create a yaml file named `object-store.yaml` with contents similar to the following example. See region to endpoint mappings [here](https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region).
+To use the Kubernetes secret method for allowing access, create a .yaml file named `object-store.yaml` with contents similar to the following example. See region to endpoint mappings [here](https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region).

```

Comment:
```suggestion
To use the Kubernetes secret method for allowing access, create a YAML file named `object-store.yaml` with contents similar to the following example. See region to endpoint mappings [here](https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region).

```

Reply:
This is one of those formatting practices that I feel can go either way, does this matter for this specific file type? I've formatted it both ways in the past.

Reply:
Whatever format you want, it just needs to be internally consistent. It's more common to use the acronym as opposed to the file extension because both `.yml` and `.yaml` are valid and conform to YAML standards.